03_Kontrollstrukturen.md

```diff
@@ -982,9 +982,9 @@ printf("%s", text2);
 Was wird ausgegeben?
 
 [[ABC123]]
-[[?]] Zur Erinnerung: Division von Ganzzahlen streicht Dezimalstellen
-[[?]] So lange kein `\n` oder Leerzzeichen geschrieben wurde, schreibt printf in der gleichen Zeile.
-[[?]] Nach einem If-Statement werden wie normal Befehle ausgeführt
+[[?]] Zur Erinnerung: im Ergebnis der Division mit Ganzzahlen entsteht ebenfalls eine ganze Zahl.
+[[?]] So lange kein `\n` oder andere Trennzeichen ausgegeben werden, gibt printf alles als eine Zeichenkette aus.
```

Comment:
Zu dem "alles als eine Zeichenkette": 
Eine Zeichenkette an sich kann ja auch über mehrere Zeilen gehen, wäre es hier nicht besser zu sagen "alles in einer Zeile aus"?

Reply:
dann dürften im Satz entweder keine weitere Trennzeichen, auch kein Leerzeichen, erwähnt werden, also  "So lange kein `\n` ausgegeben wird, gibt ..." oder den Hinweis vielleicht ganz weg lassen?

Reply:
Ja, ich denke, "So lange kein `\n` ausgegeben werden, gibt printf alles in der gleichen Zeile aus", dürfte gut sein, oder?

Reply:
Ich merge und mach es schnell selber.